src/main/java/io/jenkins/plugins/opentelemetry/job/jenkins/GraphListenerAdapterToPipelineListener.java

```diff
@@ -79,10 +82,10 @@ public final void onNewHead(FlowNode node) {
         } else if (PipelineNodeUtil.isEndParallelBlock(node)) {
             // ignore
         } else if (PipelineNodeUtil.isStartNode(node)) {
-            StepStartNode stepStartNode = (StepStartNode) node;
-            // we don't know how to surface the requested labels.
-            // ignore
-            LOGGER.log(Level.FINE, "begin node{} block " + PipelineNodeUtil.getDetailedDebugString(stepStartNode) );
+            // The pipeline step `node` (aka allocate) display name can be like "Allocate node: Start" or "Allocate node: Body : Start"
+            final String nodeDisplayName = PipelineNodeUtil.getDisplayName(node);
+            String nodeName = nodeDisplayName.matches(".*Body.*") ?  "Ready" : "Allocate";
```

Comment:
Are we at risk of fragility with NLS / Internationalization if we rely on a word in a display name? We usually can do checks looking at the type of the descriptor or the type of actions to differentiate different FlowNodes

Reply:
See https://github.com/jenkinsci/opentelemetry-plugin/pull/22/commits/1722898e7eae7e14437410f352da64134de62790